### PR TITLE
Export coordinates when in demo mode.

### LINKED ISSUE #2329: Charm left in initial position doesn't have gui annotations
The GUI location annotations for a charm left in its initial position are not saved in the resulting bundle when exported.

*Note that "no gui annotations" is a warning for `charm proof` which means that `bundletester` will fail.*

1. Go to demo.jujucharms.com
2. Add the wordpress charm and **make sure to not move the charm from its initial position**
3. commit the changes and click "export bundle".

now: 

```yaml
applications: 
  wordpress: 
    charm: "cs:trusty/wordpress-4"
    num_units: 1
    to: 
      - "0"
```

want: 

```yaml
applications: 
  limeds: 
    charm: "cs:trusty/wordpress-4"
    num_units: 1
    annotations: 
      "gui-x": 0
      "gui-y": 0
    to: 
      - "0"
```

### PR DESCRIPTION
**Pull request by kadams54**
Fixes #2329.

### CODE REVIEW

File: scripts/transpile.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ function transpile (fileList) {
     }, (err, data) => {
       console.log('Transpiling', fullPath);
       mkdirp.sync(directory);
-      const full = babel.transform(data, { plugins });
+      const full = babel.transform(data, { plugins, compact: false });
```
Comment (hatched) [inline scripts/transpile.js]: Thanks for including this fix

**Comment by kadams54** [comment]
## QA

Run the gui locally (`make run`). Deploy a charm (in demo/sandbox mode) and then export it. The YAML should have the coordinates.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6595/

**Comment by kadams54** [comment]
Test results seem spurious.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6596/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6597/

**Comment by kadams54** [comment]
@mitechie Good call; test added.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6601/

**Comment by mitechie** [comment]
:+1: ty for the update

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**